aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/fields/selectfield.html

```diff
@@ -0,0 +1,30 @@
+{% load aldryn_forms_tags i18n %}
+
+{% if field %}
+    <div class="form-group{% if field.field.required %} form-required{% endif %}{% if field.errors %} has-error{% endif %}">
+        {% block form_field_label %}
+            <label for="{{ field.auto_id }}">{{ field.label }}{% if field.field.required %}<abbr title="{% trans "Required field" %}">*</abbr>{% endif %}</label>
+        {% endblock form_field_label %}
+        {% block form_field %}
+            <select  class="form-control" name="{{ field.name }}" id="{{ field.auto_id }}">
+            {% comment %}
+                NOTE: Need to give the possibility to enable/disable te search field in the select dropdown.
```

Comment:
typo "te" -> "the". The same in multipleselectfield.html.
+ better to indent right this comment.
P.S.: yes, it doesn't refers to the code in general.